rules/rollup.go

```diff
@@ -30,61 +29,54 @@ import (
 	"github.com/m3db/m3metrics/filters"
 	"github.com/m3db/m3metrics/generated/proto/schema"
 	"github.com/m3db/m3metrics/policy"
+	"github.com/m3db/m3metrics/rules/models"
 
 	"github.com/pborman/uuid"
 )
 
 var (
-	emptyRollupTarget RollupTarget
+	emptyRollupTarget rollupTarget
 
 	errRollupRuleSnapshotIndexOutOfRange = errors.New("rollup rule snapshot index out of range")
 	errNilRollupTargetSchema             = errors.New("nil rollup target schema")
 	errNilRollupRuleSnapshotSchema       = errors.New("nil rollup rule snapshot schema")
 	errNilRollupRuleSchema               = errors.New("nil rollup rule schema")
 )
 
-// RollupTargetView is a human friendly representation of a rollup rule target at a given point in time.
-type RollupTargetView struct {
-	Name     string
-	Tags     []string
-	Policies []policy.Policy
-}
-
-func newRollupTargetView(target RollupTarget) RollupTargetView {
-	return RollupTargetView{
+func newRollupTargetView(target rollupTarget) models.RollupTargetView {
+	return models.RollupTargetView{
 		Name:     string(target.Name),
 		Tags:     stringArrayFromBytesArray(target.Tags),
 		Policies: target.Policies,
 	}
 }
 
-// RollupTarget creates a rollup target from a rollup target view.
-func (rtv RollupTargetView) RollupTarget() RollupTarget {
-	return RollupTarget{
-		Name:     []byte(rtv.Name),
-		Tags:     bytesArrayFromStringArray(rtv.Tags),
-		Policies: rtv.Policies,
+func rollupTargetViewsToTargets(views []models.RollupTargetView) []rollupTarget {
```

Comment:
Where is this function used?

Reply:
This was moved from a different location in the file for readability. But it looks like it is used in ruleset.go on line 811 and 841. I guess it could be moved to that file but I would imagine it was put here to lump it with the other rollup related functions.